### DISCUSSION TITLE
Support Environment.Action() in `pac pfx`

### DISCUSSION THREAD

**Discussion by devkeydet** [microsoft/Power-Fx#2447] [Ideas]
Any plans to support Environment.Action() in `pac pfx` similar to what can be done in canvas?

**Comment by MikeStall** [comment]
Can you share a pointer with more info?

**Reply by filcole** [reply]
I think @devkeydet is referring to this:
https://learn.microsoft.com/en-us/power-apps/maker/canvas-apps/connections/connection-common-data-service#call-dataverse-actions-directly-in-power-fx

**Reply by MikeStall** [reply]
Yes, this is coming. 

In particular, we want to ensure the e2e works where you can define an action in Power Fx via Low Code Plugins; and then call it from Power Fx in PAC CLI repl.

**Reply by chrispiasecki** [reply]
@MikeStall any update on the progress of this?
